Scripts/DCS-BIOS/lib/common/Functions.lua

```diff
@@ -1,10 +1,41 @@
+module("Functions", package.seeall)
 
----@func Returns the path of the calling script
+local Functions = {}
+
+
+---Returns the path of the calling script
 ---@return string
-function ScriptPath()
+function Functions.ScriptPath()
```

Comment:
As these are no longer global functions, their names should start with a lowercase letter correct?